yarn-project/aztec/Dockerfile

```diff
@@ -1,4 +1,6 @@
 FROM aztecprotocol/yarn-project-prod AS yarn-project-prod
+# ENV vars for using native ACVM simulation
+ENV ACVM_BINARY_PATH="/usr/src/noir/noir-repo/target/release" ACVM_WORKING_DIRECTORY="/tmp/acvm"
```

Comment:
Just a minor thing: I'd've expected the `BINARY_PATH` to point to the binary, not to the folder that contains it.